### DISCUSSION TITLE
[Feature] Sort timeline by Created and/or Updated date

### DISCUSSION THREAD

**Discussion by mmomjian** [immich-app/immich#9138] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would love the ability to view the timeline by something other than the date that the image was taken. I know the way we load the timeline makes some things, like jumping to a specific date, challenging, but I think changing the sort order should be doable. I am hoping for the ability to view images sorted by the following SQL queries:

```
SELECT * FROM "assets" ORDER BY "createdAt" DESC;
SELECT * FROM "assets" ORDER BY "updatedAt" DESC;
```

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Is this a dupe of #7489?

**Comment by mmomjian** [comment]
I don't think so. #7489 is asking to view from oldest to newest. I was hoping to get a view based on upload/update date, which is not the same as the date the picture was taken. Example: I upload a bunch of old pictures from a folder on my PC, I would like a view where I see the images that were added to Immich most recently at the top.

**Reply by bo0tzz** [reply]
Then I think it's a duplicate of #9096 :P

**Comment by mmomjian** [comment]
Agh. yes. Github search really needs some help. I'll move it over to there.
